torch/_inductor/lowering.py

```diff
@@ -1716,6 +1716,13 @@ def new_empty_strided(
     )
 
 
+@register_lowering(prims.copy_strided.default)
+def copy_strided(x, stride):
+    stride = [V.graph.sizevars.size_hint(x) for x in stride]
+    stride_order = sorted(range(len(stride)), key=stride.__getitem__)
```

Comment:
This won't work if there are gaps in the strides, but this is uncommon enough for this op I think we can just assert.